src/components/Form.js

```diff
@@ -2,25 +2,44 @@ import styled from 'styled-components/macro';
 import { useState, useEffect } from 'react';
 
 export default function Form({ notes, setNotes }) {
-  const [inputData, setInputData] = useState({ date: '', title: '', text: '' });
+  const [inputData, setInputData] = useState({ date: '', title: '', text: '', location: '' });
   const [isSubmitted, setIsSubmitted] = useState(false);
+  const [location, setLocation] = useState();
+  const [status, setStatus] = useState(null);
 
   useEffect(() => {
-    setInputData({ ...inputData, date: getDate() });
+    if (!navigator.geolocation) {
+      setStatus('Geolocation not supported!');
+    } else {
+      setStatus('Locating...');
+      navigator.geolocation.getCurrentPosition(
+        position => {
+          setTimeout(() => setStatus(null), 1000);
+          setStatus('Locating...');
+          const lat = position.coords.latitude;
+          const lng = position.coords.longitude;
+          async function fetchData() {
+            const response = await fetch(
```

Comment:
maybe you could add an error catch in case the fetch is not working out - however, might be also the next userStory😊